qutebrowser/browser/commands.py

```diff
@@ -911,12 +911,22 @@ def toggle_inspector(self):
                 cur.inspector.show()
 
     @cmdutils.register(instance='command-dispatcher', scope='window')
-    def download_page(self):
-        """Download the current page."""
-        page = self._current_widget().page()
```

Comment:
Good idea! But please keep `:download-page` for now, and add a `deprecated="Use :download instead."` to the `@cmdutils.register` decorator. This will keep it hidden everywhere, but it still works for now.